test/pylib/manager_client.py

```diff
@@ -127,13 +131,42 @@ async def before_test(self, test_case_name: str) -> None:
             # await self._wait_for_cluster()
             await self.driver_connect()  # Connect driver to new cluster
 
+    async def mark_scylla_log(self, test_case_name: str, test_log: Path) -> None:
+        """
+        Mark the place in the log file to be able to extract only part related to the test and add a handler for test
```

Comment:
I disagree with the idea of extracting only part of the log.

A failure could be related to something that happened before the test started. For example, a bug during node bootstrap, which happened during cluster setup phase, before test started using the cluster.

Reply:
We are attaching it before the test phase so it will be catched there. In the worst-case scenario, we have in the directory the full Scylla log as well.